src/main/java/com/easyrest/pages/AdministratorPage.java

```diff
@@ -13,4 +20,9 @@ public AdministratorMenu getAdministratorTabMenu() {
         return new AdministratorMenu(driver);
     }
 
+    public void waitForPagePresence() {
```

Comment:
do you need to add this method in page object?

Reply:
yes. this method waits for the presence of the administrator page